vasil-testnet/genesis/shelley.json

```diff
@@ -51,6 +51,6 @@
         "pools": {},
         "stake": {}
     },
-    "systemStart": "2022-05-19T02:00:00Z",
+    "systemStart": "2022-06-06T14:00:00Z",
     "updateQuorum": 3
 }
```

Comment:
how all these values are generated?

Reply:
This is the genesis shelley for the `vasil-dev` testnet as available [here](https://book.world.dev.cardano.org/environments.html) and [here](https://github.com/input-output-hk/cardano-configurations/tree/master/network/vasil-dev). The second link we have as a submodule in our repository and the `hydra-cluster` executable knows about them as `KnownNetwork` so if you use that, it will copy these files into the `--state-directory`. See the `README` of `hydra-cluster` for some details.